packages/ui/src/components/SelectInputV2/__tests__/index.test.tsx

```diff
@@ -780,7 +780,7 @@ describe('SelectInputV2', () => {
 
     await userEvent.keyboard('[Backspace]')
     await userEvent.keyboard('[Enter]')
-  }, 6000)
+  }, 10000)
```

Comment:
I didn't see it on SelectInputV2, but why setting a timeout? There is a general timeout for tests is it too low for select input to work?

Reply:
Yes, sometimes the default one is not enough and the test fails